core/block/export/export.go

```diff
@@ -451,6 +526,39 @@ func (e *export) handleFileObject(ctx context.Context, keys *files.FileKeys, spa
 	return keys, nil
 }
 
+func (e *export) saveFiles(ctx context.Context, b sb.SmartBlock, wr writer) {
+	st := b.NewState()
+	fileHashes := e.getFileHashes(st)
+	for _, fh := range fileHashes {
+		if _, werr := e.saveFile(ctx, wr, fh); werr != nil {
+			log.With("hash", fh).Warnf("can't save file: %v", werr)
+		}
+	}
+}
+
+func (e *export) getFilesFromRelations(st *state.State, fileHashes []string) []string {
+	for _, relLink := range st.GetRelationLinks() {
+		if relLink.Format == model.RelationFormat_file {
+			if relLink.GetKey() == bundle.RelationKeyCoverId.String() {
+				v := pbtypes.GetString(st.Details(), relLink.GetKey())
+				_, err := cid.Decode(v)
+				if err != nil {
+					// this is an exception cause coverId can contain not a file hash but color
+					continue
+				}
+			}
+			if fileHash := pbtypes.GetString(st.Details(), relLink.GetKey()); fileHash != "" {
```

Comment:
GetStringList covers the case with String value, it returns `[]string{stringValue}`